src/main/java/edu/agh/wfiis/designpatterns/patternsUsageExample/DesignPatternsManager.java

```diff
@@ -1,47 +1,31 @@
 package edu.agh.wfiis.designpatterns.patternsUsageExample;
 
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
 import java.io.File;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
 
+@Service
 public class DesignPatternsManager {
```

Comment:
Single responsibility principle is broken because class should have only a single responsibility i.e. changes to only one part of the software's specification should be able to affect the specification of the class.